test/Sentry.Tests/ModuleInit.cs

```diff
@@ -2,8 +2,17 @@
 public static class ModuleInit
 {
     [ModuleInitializer]
+    [SuppressMessage("Usage", "CA2255:The \'ModuleInitializer\' attribute should not be used in libraries")]
     public static void Init()
     {
+#if TRIMMABLE
+        Verifier.DerivePathInfo((sourceFile, projectDirectory, type, method) => new(
+            directory: projectDirectory,
+            typeName: type.Name,
+            // This ensures a unique name for our verify files when trimming is enabled
+            methodName: method.Name + ".AOT"
```

Comment:
Does this need we have to have all these new "AOT.verified" files? Do we really need that or can we get away to have these files if necessary,  the same way we do when there are variants for each .net framework version? 

As is in the current state of this PR, it's too hard to see whether there are changes in these files between AOT and non-AOT version - it would require the reviewer to individually open each of the new .verified file and compare it to the previous version committed to the repository, without that suffix.

I believe it is quite important to see whether there are differences between AOT and non-AOT outputs so I'd strongly prefer if we could not add new files unless the changes are there.

Reply:
That sounds great if we could find a way to do that. However Verify doesn't have any explicit support for AOT (in the same way that [it does for different target frameworks and architectures etc.](https://github.com/VerifyTests/Verify/blob/main/docs/naming.md#uniquefor)).

We could [follow up with Simon](https://github.com/orgs/VerifyTests/discussions/1029).

Reply:
Can we annotate test cases that need to have AOT specific golden files with a custom annotation and then check it here with `method.GetCustomAttribute` or some such?

Reply:
That should be possible yes... which would at least reduce the number of `AOT.verified` files to just those that are required/different from their managed counterparts. 

I think there's also another issue we need to address at the same time, which is that CI doesn't currently run `dotnet test SentryAot.slnf` so it doesn't check the `AOT.verified` files at the moment. That's a simple tweak to our `build.yml` workflow.